Directory.Packages.props

```diff
@@ -43,5 +43,6 @@
 		<PackageVersion Include="NSubstitute" Version="5.1.0"/>
 		<PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.9.0"/>
 		<PackageVersion Include="coverlet.collector" Version="6.0.0"/>
+		<PackageVersion Include="Microsoft.Data.SqlClient" Version="5.2.0" />
```

Comment:
Erk. FAnsiSql provides this (with a different version), once the next FAnsiSql release is pulled in can we revert this bit please?